lib/Browser.js

```diff
@@ -122,21 +122,26 @@ class Browser extends EventEmitter {
     return version.product;
   }
 
+
   async close() {
     await this._closeCallback.call(null);
     this.disconnect();
   }
 
   disconnect() {
     this._connection.dispose();
+    this.emit(Browser.Events.Disconnected, 'connection disconnected');
```

Comment:
this event should be emitted when the browser gets closed; e.g. when it crashes.

Reply:
Done, let me know if it still wrong. So it's introduce breaking changes?

Reply:
Wait, I think there's some miscommunication here.

I actually meant the browser crash, not the page crash.

The only signal we get when the browser crashes is the disconnect of web socket (see lib/Connection.js). We should capture this event and dispatch the `disconnected` for it

Reply:
Okay, we take a look on it. I also need your advice to reflect the documentation changes for this one. FYI build fail because travis are down.

Reply:
I hope that was the right one, i skip the test for this case.